x-pack/plugins/security_solution/public/flyout/preview/panels.tsx

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import type { PreviewPanelPaths } from '.';
+import { RULE_PREVIEW } from './translations';
+import { RulePreview } from './components/rule_preview';
+import { RulePreviewFooter } from './components/rule_preview_footer';
+
+export type PreviewPanelType = Array<{
```

Comment:
let's add a brief documentation for each property

Reply:
I think this is self-explanatory and matches what we have in left and right sections. I can still add documentation, but it's going to be "id of preview panel", "content of preview panel", wdyt?